config/samples/nvidia_v1alpha1_nvidiadriver.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   driverType: gpu
   repository: nvcr.io/nvidia
   image: driver
-  version: "550.144.03"
+  version: "570.86.15"
```

Comment:
@cdesiniotis you mean here?

Reply:
No, I mean here: https://github.com/NVIDIA/gpu-operator/blob/3e33cd9a87a5dbada410485a884deeea7d9a2fa5/bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml#L164

Reply:
Ah, thanks for catching that!

Reply:
Done